Classes/Player.ts

```diff
@@ -0,0 +1,17 @@
+import {greetingDisp} from "../selectors/selectors.js";
```

Comment:
Da du den Selector nur in Player verwendest, würd ich den auch in Player schreiben.

Wenn du im JavaScript Kurs weiter bist wird auch erklärt, wie man sowas schön umsetzen kann:
https://wesbos.com/javascript/10-harder-practice-exercises/58-building-a-gallery

Wenn du soweit bist, dann schreib diesen Code nochmal um, dass `Player` ein HTML-Element im Constructor übergeben bekommt, und alles innerhalb kontrolliert. Der Selector wär dann `this.element.querySelector('.greeting');`